src/virtual_elements.js

```diff
@@ -285,7 +285,7 @@ var elementVoid = function(tag, key, statics, var_args) {
  * Declares a virtual Text at this point in the document.
  *
  * @param {string|number|boolean} value The value of the Text.
- * @param {...(function(string|number|boolean):string)} var_args
+ * @param {...function((string|number|boolean)):string} var_args
```

Comment:
Ok this must be a problem with either or both gjslint and closure then because you get this currently
```
Viewport.js:85: WARNING - Function module$idom$index.text: called with 1 argument(s). Function requires at least 2 argument(s) and no more than 2 argument(s).
    text(txt)
    ^
idom/src/virtual_elements.js:288: WARNING - Bad type annotation. missing closing )
 * @param {...(function(string|number|boolean):string)} var_args
```